### PR TITLE
Slack initiated update, typo and rewrite-Update backup_intro.adoc

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

File: backup_restore/backup_intro.adoc [hunk 1]
```diff
@@ -222,4 +222,4 @@ spec:
   disableHubSelfManagement: true
 ----
 
-*Note:* If the self managing option is not disabled on the restore hub cluster before the activation data is moved to the restore hub cluster, the `local-cluster` klusterlet and the klusterlet in the managed cluster namespace conflict. As a result, the restore hub cluster is managed by itself by using the managed cluster and by using the restored managed cluster. If you detach the managed cluster as part of the detach operation, the managed cluster receives a deprovision request, which results in the restore hub cluster rmeoving itself.
+*Note:* If the self managing option is not disabled on the restore hub cluster before the activation data is moved to the restore hub cluster, the `local-cluster` klusterlet and the klusterlet in the managed cluster namespace conflict. As a result, the restore hub cluster is managed by itself by using the managed cluster and by using the restored managed cluster. If you detach the managed cluster as part of the detach operation, the managed cluster receives a deprovision request, which results in restore hub cluster removal.
```
Comment (dockerymick) [inline backup_restore/backup_intro.adoc:225]: which results in restoring the hub cluster removal

**Review by dockerymick — APPROVED** [review]
Made a suggestion, otherwise looks good to me

File: backup_restore/backup_intro.adoc (hunk 1 above)
Comment (swopebe) [reply backup_restore/backup_intro.adoc:225]: I think it is a "restore hub cluster" ??? (Restore describing the cluster)

I deduced this based on surrounding content.

Comment (swopebe) [reply backup_restore/backup_intro.adoc:225]: Example: data is moved to the restore hub cluster

**Pull request merged**